dag-syntax.cddl

```diff
@@ -0,0 +1,24 @@
+application = {
+    applicationName: tstr,
+    nodes: [ * node ],
+}
+
+node = {
+    task: tstr,
+    name: tstr,
+    type: "producer" / "prosumer" / "consumer",
+    description: tstr,
+    executor: "container",
```

Comment:
`executor` should have an alternative possible value called `static` for unscalable, unmovable instances that run directly on a given node (e.g. the Iceflow Edge Interface).

Reply:
```suggestion
    executor: "container" / "static",
```